fuzz/test-corpus.c

```diff
@@ -16,31 +16,82 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <sys/stat.h>
 #include <openssl/crypto.h>
 #include "fuzzer.h"
+#include "internal/o_dir.h"
+
+#ifdef __WIN32
```

Comment:
MSCV doesn't define __WIN32, it's _WIN32.

Reply:
I should probably check if _MAX_PATH is defined rather than platform anyway...